aead/src/lib.rs

```diff
@@ -299,16 +278,30 @@ impl<Alg: AeadInPlace> Aead for Alg {
     }
 }
 
-#[cfg(feature = "inout")]
-impl<T: AeadInOut + PostfixTagged> AeadInPlace for T {
+impl<T: AeadInOut> Aead for T {
```

Comment:
I guess `aes_siv` should not implement `AeadInOut`?

This blanket doesn't work for Siv. (https://github.com/RustCrypto/AEADs/pull/674)

Reply:
It's a bug in the blanket impl. I will fix it shortly.